src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -157,13 +157,12 @@ public enum OperationKind
 
         // Operations that are constituents of statements, expressions, or declarations.
 
-        /// <summary>Indicates an <see cref="IFieldInitializer"/>.</summary>
-        FieldInitializerInCreation = 0x400,
+
+        // Unused 0x400 and 0x402
+
         /// <summary>Indicates an <see cref="IFieldInitializer"/>.</summary>
         FieldInitializerAtDeclaration = 0x401,
```

Comment:
Sounds like a good idea. Any objections from others? Otherwise, I will make this change.